_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: 4e8ab6
-name: Role has required states and properties
+name: Element with role has required states and properties
```

Comment:
Backsticks and 'attribute' word no ? 

> Element with *`role` attribute* ?

Reply:
In this case I believe the role is implicit rather than an attribute

Reply:
Apparently not ... this rule checks elements with the role attribute.